brian2/input/timedarray.py

```diff
@@ -82,6 +83,8 @@ class TimedArray(Function, Nameable):
      [ 1.  3.]
      [ 2.  4.]] mV
     '''
+    _cache_irrelevant_attributes = {'_id', 'values', 'pyfunc',
+                                    'implementations'}
```

Comment:
This line could be a little risky if we decide to modify cacheing to be persistent across sessions? I mean specifically ``implementations`` (although maybe others). If this is specific to this case, we should consider changing, but if we're universally doing this then adding this one is no problem.